packages/sol6/src/yang/etsi-nfv-vnf.yang

```diff
@@ -1664,16 +1664,23 @@ submodule etsi-nfv-vnf {
               "GS NFV IFA011: Section 7.1.5.2
                VnfLcmOperationsConfiguration information element";
 
-            leaf parameter {
-              type string;
+            list parameter {
+              key "key";
+              leaf key {
+                type string;
+              }
+              leaf value {
+                type string;
+              }
               description
                 "VNF-specific parameter to be passed when invoking
-                 the HealVnf operation.";
+                 the ScaleVnfToLevel operation.";
```

Comment:
This should read as HealVnfOpConfig. The comment should also reflect that it is now a KVP.

Reply:
Copy past, sorry!